lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1710,6 +1710,18 @@ def test(self):
         cube = stock.simple_3d()
         self._check_copy(cube, cube.copy())
 
+    def test_copy_ancillary_variables(self):
+        cube = stock.simple_3d()
+        avr = AncillaryVariable([2, 3], long_name="foo")
+        cube.add_ancillary_variable(avr, 0)
+        self._check_copy(cube, cube.copy())
+
+    def test_copy_cell_measures(self):
+        cube = stock.simple_3d()
+        cms = CellMeasure([2, 3], measure="area", long_name="foo")
```

Comment:
Since #3533 "area" is now the default for `measure` and this keyword argument has therefore been removed from all tests